stdlib/3/builtins.pyi

```diff
@@ -50,7 +50,7 @@ class object:
     def __sizeof__(self) -> int: ...
     def __reduce__(self) -> tuple: ...
     def __reduce_ex__(self, protocol: int) -> tuple: ...
-    def __dir__(self) -> List[str]: ...
+    def __dir__(self) -> _Collection[str]: ...
```

Comment:
Should this just be `Iterable`? Empirically, `dir()` is happy with a generator.

Reply:
Agreed, I was being too clever. Done.